### PR TITLE
Docs/poc/pg4kedbrepohappypath

### PR DESCRIPTION
**Pull request by djw-m**
Signed-off-by: Dj Walker-Morgan <dj.walker-morgan@enterprisedb.com>

## What Changed?

Installation_Upgrade rework to inform changes to maint (or to force an override)

### CODE REVIEW

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 1]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
 
 ### Directly using the operator manifest
 
+#### Install the EDB pull secret
+
+Before installing EDB Postgres for Kubernetes, you need to create a pull secret for EDB software in the `postgresql-operator-system` namespace.
+
+
+The pull secret needs to be save in the namespace where the operator will reside. Create a postgresql-operator-system namespace using this command:
+
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:56]:
```suggestion
The pull secret needs to be saved in the namespace where the operator will reside. Create a postgresql-operator-system namespace using this command:

```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 2]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
+
+Export the appropriate value as an environmental variable EDB_SUBSCRIPTION_PLAN.
+
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:29]:
```suggestion

Export the appropriate value as an environment variable EDB_SUBSCRIPTION_PLAN.

```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 3]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
 
-You can install the [latest operator manifest](https://get.enterprisedb.io/cnp/postgresql-operator-1.24.1.yaml)
-for this minor release as follows:
+You can install the manifest for the latest version of the operator by running:
 
 ```sh
 kubectl apply --server-side -f \
-  https://get.enterprisedb.io/cnp/postgresql-operator-1.24.1.yaml
+  https://get.enterprisedb.io/pg4k/pg4k-enterprise-1.24.1.yaml
+
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:83]:
```suggestion
  "https://get.enterprisedb.io/pg4k/pg4k-${EDB_SUBSCRIPTION_PLAN}-1.24.1.yaml"

```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 4]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+kubectl create namespace postgresql-operator-system
+```
+
+To create the pull secret itself, run the following command:
+
+```shell
+kubectl create secret -n postgresql-operator-system docker-registry edb-pull-secret \
+  --docker-server=docker.enterprisedb.com \
+  --docker-username=k8s_$EDB_SUBSCRIPTION_PLAN \
+  --docker-password=$EDB_SUBSCRIPTION_TOKEN
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:67]:
```suggestion
  --docker-username="k8s_${EDB_SUBSCRIPTION_PLAN}" \
  --docker-password="${EDB_SUBSCRIPTION_TOKEN}"
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 5]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
+
+Export the appropriate value as an environmental variable EDB_SUBSCRIPTION_PLAN.
+
+```shell
+export EDB_SUBSCRIPTION_PLAN=enterprise
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:31]: You don't need to export the variables to use them in the same shell.

```suggestion
EDB_SUBSCRIPTION_PLAN=enterprise
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 6]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+```
+
+then export the Repos 2.0 token to an environment variable EDB_SUBSCRIPTION_TOKEN.
+
+```shell
+export EDB_SUBSCRIPTION_TOKEN=<your-token>
```
Comment (mnencia) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:37]: You don't need to export the variables to use them in the same shell.

Suggested change
```suggestion
EDB_SUBSCRIPTION_TOKEN=<your-token>
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 7]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
```
Comment (jsilvela) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:26]: both include the standard operands, as well as the operators. Worth mentioning?

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 8]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+
 ### Using the Helm Chart
 
 The operator can be installed using the provided [Helm chart](https://github.com/EnterpriseDB/edb-postgres-for-kubernetes-charts).
 
 ### Directly using the operator manifest
 
+#### Install the EDB pull secret
+
+Before installing EDB Postgres for Kubernetes, you need to create a pull secret for EDB software in the `postgresql-operator-system` namespace.
```
Comment (jsilvela) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:52]: I know you already mention "EDB software" in the section on getting the token, but I think this paragraph is clearer if saying the pull secret gives access to the private registries. I.e. not to "EDB software" in general.

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 9]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
 
 ## Installation on Kubernetes
 
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
```
Comment (djw-m) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:20]:
```suggestion
You can obtain the token by visiting your [EDB Repos account page](https://www.enterprisedb.com/accounts/progile) and signing into your account.
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 10]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
 
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting your [EDB Repos account page](https://www.enterprisedb.com/accounts/progile) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
```
Comment (djw-m) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:25]:
```suggestion
* Standard - Includes the operator and standard operands, including EDB Postgres Extended.
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx [hunk 11]
```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
...
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
```
Comment (djw-m) [inline product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:26]:
```suggestion
* Enterprise - Includes the operator and standard and enterprise operands, including EDB Postgres Advanced and EDB Postgres Extended.
```

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx (hunk 7 above)
Comment (djw-m) [reply product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:26]: Applied a tweak to the text.

File: product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx (hunk 8 above)
Comment (djw-m) [reply product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx:52]:
```suggestion
Before installing EDB Postgres for Kubernetes, you need to create a pull secret for EDB private registries in the `postgresql-operator-system` namespace.
```

**Pull request closed without merging**